perceval/backends/core/github.py

```diff
@@ -389,18 +389,40 @@ def __get_pull_requested_reviewers(self, pr_number):
         return requested_reviewers
 
     def __get_pull_commits(self, pr_number):
-        """Get pull request commit hashes"""
+        """Get pull request commit info"""
```

Comment:
`info` seems to be too generic, you could have something like:
`"""Get pull request commit hashes and comments"""`